database/migrations/2023_06_24_093715_create_accomodations_table.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('accomodations', function (Blueprint $table) {
+            $table->id();
+            $table->string('name');
+            $table->string('country');
+            $table->string('city');
+            $table->string('quarter');
+            $table->longText('description');
+            $table->boolean('available')->default(true);
+            $table->timestamps();
```

Comment:
All of these are supposed to be rooms in the same building but I think you can allow these fields.

Add `accommodation_number` and make everything nullable except for `name` and `accommodation_number`